StRoot/StFstRawHitMaker/StFstRawHitMaker.h

```diff
@@ -0,0 +1,97 @@
+#ifndef StFstRawHitMaker_hh
+#define StFstRawHitMaker_hh
+
+#ifndef __CINT__
+#include <array>
+#include <iostream>
+#endif
+
+#include <vector>
+
+#include "St_base/Stypes.h"
+#include "StChain/StRTSBaseMaker.h"
+#include "StEvent/StFstConsts.h"
+
+
+class StFstCollection;
+
+
+/**
+ * Reads/un-packs a DAQ or SFS file; Accesses calibration DBs;
+ * Marks out bad channels/chips; Creates and fills the FST containers.
+ * Two working modes included:
+ * 1) calibration mode: save raw ADC value only
+ * 2) non-calibration (physics) mode: access calibration datasets, signal-like
+ *    raw hit decision, full raw hit info., ...
+ *
+ * \Author Shenghui Zhang
+ * \date Aug 2021
+ */
+class StFstRawHitMaker : public StRTSBaseMaker
+{
+public:
+   StFstRawHitMaker( const char *name = "fst_raw_hit" );
+   ~StFstRawHitMaker();
+   Int_t Init();
+   Int_t InitRun(Int_t runNumber);
+   Int_t Make();
+   void Clear( Option_t *opts = "" );
+
+   void setIsCalibrationMode( bool isCaliMode = false )  { mIsCaliMode = isCaliMode; }
+   void setMinHitCut(double minhitCut = 2.5)             { mMinHitCut = minhitCut; }
+   void setMedHitCut(double medhitCut = 3.5)             { mMedHitCut = medhitCut; }
+   void setMaxHitCut(double maxhitCut = 4.0)             { mMaxHitCut = maxhitCut; }
+   void setCmnCorrection( bool doCmnCorrection = true )	 { mDoCmnCorrection = doCmnCorrection; }
+   void setCmnCut(double cmnCut = 3.)                    { mCmnCut = cmnCut; }
+   /// 0 - All data; 1 - non-ZS data; 2 - ZS data; 3 - ZS first data
+   void setDataType(int nDataType = 0)                   { mDataType = nDataType; };
+   void setDoEmbedding(Bool_t doIt)                      { mDoEmbedding = doIt; }
+   UChar_t getDataType() {return mDataType;} // 0 - All data; 1 - non-ZS data; 2 - ZS data; 3 - ZS first data
+
+   // Get CVS
+   virtual const char *GetCVS() const {
+      static const char cvs[] = "Tag $Name: SL20a $ $Id: StFstRawHitMaker.h,v 1.26 2018/02/25 03:51:57 dongx Exp $ built " __DATE__ " " __TIME__  ;
+      return cvs;
+   }
+
```

Comment:
We do not need the CVS string any more.

Reply:
removed in a1b656422171901b3c0f9af39228875d9fe1be5e